keyboards/converter/thinkpad_t6x/rpi_pico/readme.md

```diff
@@ -0,0 +1,27 @@
+# converter/thinkpad_t6x/rpi_pico
+
+![converter/thinkpad_t6x/rpi_pico](https://github.com/strobo5/T61_PiPico_Scanner/blob/a6e127c8824a4aa1ad8d5b643b36fcde0087b1c5/images/kb.jpg?raw=true)
```

Comment:
Please rehost this image on Imgur. Github urls are prone to changing.

Reply:
I tried, but I think I failed to get the link right. Help? :sweat_smile: This is my image https://imgur.com/a/Tb09udi

Reply:
ah I didn't see fauxpax's suggestion, nvm :)